packages/test/local-server-stress-tests/src/test/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": "../../../../../common/build/build-common/tsconfig.test.node16.json",
+	"compilerOptions": {
+		"rootDir": "../",
```

Comment:
rootDir and outDir look unusual for test tsconfigs--`"./"` and `"../../lib/test"` are the values I see elsewhere in the repo for split builds.

If your intent is to avoid having a separate tsconfig for tests of this package since the whole package is a test package, I think moving this tsconfig up a level in hierarchy and using `"rootDir": "./"` and `"outDir": "../lib"` would make more sense.

Reply:
i just copied the existing local-server-tests package and used it as scaffolding, not sure what the right why to do anything is. happy to make changes if you think another way is better

Reply:
Ehh. I guess `local-server-tests` and `test-end-to-end-tests` follow the pattern you're using now. So probably no need for changes now. I think having a tsconfig with a rootDir that actually navigates up the source tree is pretty weird and would prefer all of them moved their tsconfig along the lines of my suggestion. Consistency between the three is more important though, so probably just leave as-is.